internal/platform/time_test.go

```diff
@@ -44,25 +43,10 @@ func Test_Walltime(t *testing.T) {
 }
 
 func Test_Nanotime(t *testing.T) {
-	tests := []struct {
-		name     string
-		nanotime sys.Nanotime
-	}{
-		{"Nanotime", Nanotime},
-		{"nanotimePortable", nanotimePortable},
-	}
-
-	for _, tt := range tests {
-		tc := tt
-		t.Run(tc.name, func(t *testing.T) {
-			delta := time.Since(nanoBase).Nanoseconds()
-			nanos := Nanotime()
-
-			// It takes more than a nanosecond to make the two clock readings required
-			// to implement time.Now. Hence, delta will always be less than nanos.
-			require.True(t, delta <= nanos)
-		})
-	}
+	n1 := Nanotime()
+	time.Sleep(time.Second)
```

Comment:
seems a bit long, as this test runs all the time, if it is nanos precision can we do a millisecond or 10?